### DISCUSSION TITLE
When Export Pdf  (arabic Language ) Problem in Letters And Words With Arabic. How Solve it  JsPdf?

### DISCUSSION THREAD

**Discussion by ra-bedo11** [serenity-is/Serenity#7220] [Help]
Plz
When Export Pdf  (arabic Language ) Proplem in Letters And Words With Arabic ????? How Solve it  JsPdf

![serenityArabic](https://github.com/user-attachments/assets/30b3bf42-f6d9-4fad-aa8b-a52567c1fa6f)

**Comment by VictorTomaili** [comment]
ask this in jspdf repository.

**Reply by ra-bedo11** [reply]
where jspdf repository. ? How

**Reply by asilmertkan** [reply]
JsPDF is third party library, so you can search https://github.com/parallax/jsPDF in there and try to help from these community ;)
